### PR TITLE
fix nav for newish documents - Fix #4080

### LINKED ISSUE #4080: Docs deploy?
Seems the docs deploy failed on the 4.71 deploy - I can't find the BioNano Genomics stuff on GitHub.io?

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Docs fixes only. Much better on local at least now:

Screenshot 2023-09-19 at 16 34 20

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
Fix #4080!

File: docs/admin-guide/bionano_access_integration.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Matchmaker Exchange integration
+## BioNano Access Server Integration
```
Comment (northwestwitch) [inline docs/admin-guide/bionano_access_integration.md:1]: 🤣

File: mkdocs.yml [hunk 2]
```diff
@@ -59,19 +59,23 @@ nav:
...
       - PanelApp gene panels: 'admin-guide/panelapp_panels.md'
-      - Gens integration: 'admin-guide/gens_integration.md'
       - HPO terms: 'admin-guide/hpo.md'
       - Annotations: 'admin-guide/annotations.md'
       - IGV browser settings: 'admin-guide/igv-settings.md'
       - Server: 'admin-guide/server.md'
-      - Chanjo coverage integration: 'admin-guide/chanjo_coverage_integration.md'
-      - Loqusdb integration: 'admin-guide/loqusdb_integration.md'
-      - Matchmaker Exchange integration: 'admin-guide/matchmaker_exchange_integration.md'
+      - Server Extension Integration:
```
Comment (northwestwitch) [inline mkdocs.yml:66]: 💯

**Comment by dnil** [comment]
Ah, ok, we also had a settings issue. Note that we have this error:

Googling leads to changing the source folder to publish out of on the `gh-pages` branch to root instead of docs (which does not seem to be there?)

Screenshot 2023-09-19 at 16 55 22

That would probably mean its been a little while since those updated last?

**Review by northwestwitch — COMMENTED** [review]
Great fixes! 

There's a couple of warnings still:

```
WARNING -  Doc file 'admin-guide/containers/systemd.md' contains a relative link './scout.fcc', but the target 'admin-guide/containers/scout.fcc' is not found
           among documentation files.
WARNING -  Doc file 'user-guide/genes.md' contains a relative link 'www.genenames.org', but the target 'user-guide/www.genenames.org' is not found among
           documentation files.
```

Can they be fixed?

**Comment by dnil** [comment]
ahem, yes, that worked almost too well - this branch is now already live on the site.. 😊

**Comment by dnil** [comment]
The "scout.fcc" thing is since the docs file contains a link to the main repo. The `gh-pages` branch does not contain a copy of the source code. We could fix it either by including a static copy of the `.fcc` in the docs, or by using like a permalink to GitHub instead?

**Comment by northwestwitch** [comment]
> The "scout.fcc" thing is since the docs file contains a link to the main repo. The `gh-pages` branch does not contain a copy of the source code. We could fix it either by including a static copy of the `.fcc` in the docs, or by using like a permalink to GitHub instead?

The second option seems easier

**Comment by dnil** [comment]
> The second option seems easier

Yep, the fcc was fairly large. I opted on making it a just a path in text instead - I think that works even better. The permalink would be blobbed to the current version, which doesn't make a lot of sense.

**Review by northwestwitch — APPROVED** [review]
I'm approving since this fixes the issue. The problems before can be also fixed in another PR. Good job!

**Comment by dnil** [comment]
> I'm approving since this fixes the issue. The problems before can be also fixed in another PR. Good job!

I think it made good sense to fix a few more when we were at it! All warnings are now gone, and most of the INFO points, but I chickened on two remaining INFO level un-navlinked docs that should probably just be deleted - or rewritten, and include links to other repos as well.

**Comment by northwestwitch** [comment]
> I think it made good sense to fix a few more when we were at it! All warnings are now gone, and most of the INFO points, but I chickened on two remaining INFO level un-navlinked docs that should probably just be deleted - or rewritten, and include links to other repos as well.

I thinks it's fine already without warnings. Perhaps we could add a github action that checks if the doc links are fine, as in atlas.

**Pull request merged**